src/games/strategy/triplea/TripleA.java

```diff
@@ -115,18 +117,25 @@ public void startGame(final IGame game, final Set<IGamePlayer> players, final bo
       connectPlayers(players, null);
     } else {
       SwingAction.invokeAndWait(() -> {
-        final TripleAFrame frame;
-        frame = new TripleAFrame(game, localPlayers);
-        display = new TripleADisplay(frame);
-        game.addDisplay(display);
-        soundChannel = new DefaultSoundChannel(localPlayers);
-        game.addSoundChannel(soundChannel);
-        frame.setSize(700, 400);
-        frame.setVisible(true);
-        ClipPlayer.play(SoundPath.CLIP_GAME_START);
-        connectPlayers(players, frame);
-        frame.setExtendedState(Frame.MAXIMIZED_BOTH);
-        frame.toFront();
+        try {
+          final TripleAFrame frame;
+          frame = new TripleAFrame(game, localPlayers);
+          display = new TripleADisplay(frame);
+          game.addDisplay(display);
+          soundChannel = new DefaultSoundChannel(localPlayers);
+          game.addSoundChannel(soundChannel);
+          frame.setSize(700, 400);
+          frame.setVisible(true);
+          ClipPlayer.play(SoundPath.CLIP_GAME_START);
+          connectPlayers(players, frame);
+          frame.setExtendedState(Frame.MAXIMIZED_BOTH);
+          frame.toFront();
+        } catch (Exception e) {
+          if(game instanceof ClientGame){
```

Comment:
You're absolutely right, 
This was needed before I came up with the runnable solution...